Packs/Axonius/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Axonius
+Modifying the integration to support pulling an api-key and api-secret from credentials.
+Bumping the docker version to use our latest api client.
```

Comment:
```suggestion
- Modifying the integration configuration params to use *API Key* and *API secret* instead of *ax_key* and *ax_secret*.
- Updated the Docker image to: *demisto/axonius:1.0.0.40908*.
```